api_tests/sanity/lnd.ts

```diff
@@ -0,0 +1,21 @@
+import { twoActorTest } from "../lib_sdk/actor_test";
+
+setTimeout(function() {
+    twoActorTest("sanity-lnd-alice-pays-bob", async function({ alice, bob }) {
+        await alice.startLnd();
+        await bob.startLnd();
```

Comment:
This could be a performance issue as starting LND nodes takes some time. 
Ideally we can configure the need for LND nodes in the `config.toml` file which would start LND nodes for all tests within a particular folder. 

We can optimize this later though, just wanted to point it out.

Reply:
Paraphrasing to confirm I got you, optimise so that all tests in a single directory use the same LND instances.  Nice idea, didn't think of that myself.  FTR LND nodes must appear in the `config.toml`.  Also, I'm not totally understanding what is old test code and what is new test code as far as the harness goes (i.e., what is going away and what is staying).

Reply:
Yes, correct. Two (one for Alice and one for Bob) LND nodes per directory. 
Right now harness.ts it taking care of starting the blockchain nodes which I think we want to change later on.